index.html

```diff
@@ -30,6 +30,126 @@ <h1>Hey there. I'm John Buhendwa <span>I'm a software developer</span></h1>
                 <li><a href="#"><i class="fa fa-angellist"></i></a></li>
             </ul>
         </section>
+        <div class="button-section">
+            <button><i class="fa fa-angle-down" id="next-section"></i></button>
+        </div>
+        <section class="work-section">
+            <h1>My Recent Works</h1>
+            <div id="hr-line"></div>
+            <div class="project-container">
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
+                        </ul>
+                        <button>See Project</button>
+                    </div>
+                </div>
+                <div class="work">
+                    <div class="project-info">
+                        <h2>Multi-Post Stories Gain+Glory</h2>
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>css</li>
+                            <li>JavScript</li>
+                            <li>html</li>
```

Comment:
- [optional] you can make your tech buttons the same height as that of the Figma version. This will make your work more presentable. Nice work 
yours
![Screenshot (745)](https://user-images.githubusercontent.com/65298048/212285884-9f1e36c6-c265-4a5d-a2c6-b78ae80fe649.png)
figma
![Screenshot (744)](https://user-images.githubusercontent.com/65298048/212285991-f3f5f06b-7bf6-4fae-984b-ef62b46e84c1.png)